src/components/AddItems.js

```diff
@@ -56,6 +81,11 @@ const AddItemsToList = () => {
       <div>
         <form onSubmit={submitShoppingListItemHandler}>
           <h2>Add Item to List</h2>
+          {shoppingListItemNameExists ? (
+            <p>
+              {`You have ${shoppingListItemName} in your shopping list already`}
+            </p>
+          ) : null}
```

Comment:
I love how you display the message to the user. 👌🏻
I have one tiny suggestion: 

Currently user sees the item name that is not normalized since we are using `shoppingListItemName` on line 86. 
We could improve the user experience by showing the item name that is normalized.

To achieve this, you will need to move the `normalizeString` function outside the `submitShoppingListItemHandler` function. This way you can re-use the `normalizeString` function inside render. Therefore you can do: `normalizeString(shoppingListItemName)` on line 86. 
I think it is also a good practice to define such functions outside the scope for reusability.

Reply:
Great, thanks @Lukabaramishvili! I've made the changes, I had to change the regex slightly as the normalizeString method also removed all spaces when doing the duplicate check, so I've left these in when displaying the message to the user (eg, it would have shown redpeppers rather than red peppers). @nibble0101 this makes the regex a bit more complicated as I had to split out replacing the non word characters and duplicate spaces separately, there may be a simpler way of doing it!